tests/phantom.callback.js

```diff
@@ -1,14 +1,21 @@
 /* global page */
+/*eslint prefer-spread: 1 */
```

Comment:
Note: 4e36da24a8a20009b351a982972e3601d3d3e177 disabled `prefer-spread` rule in `tests` dir (as it should have been done from the start), so this line isn't needed any more.